lib/ansible/modules/cloud/amazon/elb_application_lb.py

```diff
@@ -791,21 +946,22 @@ def main():
     argument_spec = ec2_argument_spec()
     argument_spec.update(
         dict(
-            access_logs_enabled=dict(required=False, type='bool'),
-            access_logs_s3_bucket=dict(required=False, type='str'),
-            access_logs_s3_prefix=dict(required=False, type='str'),
-            deletion_protection=dict(required=False, default=False, type='bool'),
-            idle_timeout=dict(required=False, type='int'),
-            listeners=dict(required=False, type='list'),
+            access_logs_enabled=dict(type='bool'),
+            access_logs_s3_bucket=dict(type='str'),
+            access_logs_s3_prefix=dict(type='str'),
+            deletion_protection=dict(default=False, type='bool'),
+            idle_timeout=dict(type='int'),
+            listeners=dict(type='list'),
```

Comment:
@s-hertel see above.  Don't want to hassle you but i want to fix this up and get it merged because it's a pretty big change that i want in before 2.4 ships